### PR TITLE
[rsig] Adds new port rsig v0.1.0

### PR DESCRIPTION
**Pull request by rioki**
**Describe the pull request**

- #### What does your PR fix?
  Adds rsig; a small signal handling library.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline]
  all

- #### Does your PR follow the [maintainer guide]
  By the best of my knowledge, yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  yes (thank you for reminding me ^_^)

### CODE REVIEW

File: ports/rsig/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+file(COPY ${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt DESTINATION ${SOURCE_PATH})
+
+vcpkg_cmake_configure(
+    SOURCE_PATH ${SOURCE_PATH}
+)
+
+vcpkg_cmake_install()
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
```
Comment (Thomas1664) [inline ports/rsig/portfile.cmake:16]: You can simply use `file(INSTALL ${SOURCE_PATH}/rsig/rsig.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/rsig)`. Then you can get rid of `CMakeLists.txt` and you don't need the dependencies `vcpkg-cmake` and `vcpkg-cmake-config` anymore

Comment (rioki) [reply ports/rsig/portfile.cmake:16]: That makes sense... I went off of previous packages and simply removed the bits I don't need. Will do.

**Comment by strega-nil-ms** [comment]
Thanks!

**Comment by rioki** [comment]
No, thank you!

**Pull request merged**